packages/expo-file-system/ios/EXFileSystem/EXFileSystem.m

```diff
@@ -16,35 +15,25 @@
 
 #import <UMCore/UMEventEmitterService.h>
 
-NSString * const EXDownloadProgressEventName = @"Exponent.downloadProgress";
-
-@interface EXDownloadResumable : NSObject
-
-@property (nonatomic, strong) NSString *uuid;
-@property (nonatomic, strong) NSURLSession *session;
-@property (nonatomic, strong) EXDownloadDelegate *delegate;
+#import <EXFileSystem/EXSessionDownloadTaskDelegate.h>
+#import <EXFileSystem/EXSessionUploadTaskDelegate.h>
 
-@end
+NSString * const EXDownloadProgressEventName = @"Exponent.downloadProgress";
```

Comment:
```suggestion
NSString * const EXDownloadProgressEventName = @"Expo.downloadProgress";
```

I'm almost 100% sure that we migrated from `Exponent` to `Expo`, but still I thing this event don't need any specific expo-related prefix and might be just `downloadProgress` 🤔 I'm curious what others think though 😉